test/libsinsp/libsinsp_test_var.h.in

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#pragma once
+
+// Absolute path to the kernel module file
+#define LIBSINSP_TEST_KERNEL_MODULE_PATH "${CMAKE_BINARY_DIR}/driver/scap.ko"
+
+// Kernel module name
+#define LIBSINSP_TEST_KERNEL_MODULE_NAME "${DRIVER_NAME}"
+
+// Absolute path to the bpf probe .o file
+#define LIBSINSP_TEST_BPF_PROBE_PATH "${CMAKE_BINARY_DIR}/driver/bpf/probe.o"
+
+#define LIBSINSP_TEST_CAPTURES_PATH "${CMAKE_BINARY_DIR}/test/libsinsp/captures"
```

Comment:
`LIBSINSP_TEST_CAPTURES_PATH` has a different motivation: each test dumps the captured events in a scap file so that we can inspect them in a second moment. This is not meant to test scap file correctness. 😅